modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1515,17 +1515,31 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
   /**
    * Get the current market price from a third party to be used for recovery
    */
-  getRecoveryMarketPrice(): Bluebird<string> {
+   getRecoveryMarketPrice(): Bluebird<string> {
     const self = this;
     return co<string>(function *getRecoveryMarketPrice() {
-      const bitcoinAverageUrl = config.bitcoinAverageBaseUrl + self.getFamily().toUpperCase() + 'USD';
-      const response = yield request.get(bitcoinAverageUrl).retry(2).result();
-
-      if (response === null || typeof response.last !== 'number') {
-        throw new Error('unable to reach BitcoinAverage for price data');
+      // TODO This map should be moved to the place where the list of families is defined
```

Comment:
good thinking, but for simplicity's sake I'm fine with leaving the map close to the place it's being used.